upgrade-2.x.sh

```diff
@@ -257,8 +257,8 @@ function upgrade_supervisor() {
 
 function error_handler() {
     # If script fails (e.g. docker pull fails), restart the stopped services like the supervisor
-    systemctl start resin-supervisor
-    systemctl start update-resin-supervisor.timer
+    systemctl start balena-supervisor resin-supervisor || true
+    systemctl start update-balena-supervisor.timer update-resin-supervisor.timer || true
```

Comment:
`errexit` is set so we can't allow an exit code 1 before the last service is restarted.